tests/test_runner/test_parallel.py

```diff
@@ -220,6 +223,48 @@ def test_add_duration(self):
         self.assertEqual(result.collectedDurations, [("None", 2.3)])
 
 
+@mock.patch("django.test.runner.connections")
+@mock.patch("django.test.runner.call_command")
+@mock.patch("django.test.runner.setup_test_environment")
+@mock.patch("django.setup")
+@mock.patch("multiprocessing.get_start_method")
+class InitWorkerTests(SimpleTestCase):
+    def test_calls_setup_functions_when_start_method_is_spawn(self, *mocks):
+        mocks[0].return_value = "spawn"
+
+        counter = multiprocessing.Value(ctypes.c_int, 0)
+        initial_settings = {"default": mock.MagicMock()}
```

Comment:
This inner mock seems to be unused, perhaps an empty dict is fine?

Reply:
Thanks! Fixed in 1384454d99.